docs/application/native/guides/internationalization/i18n.md

```diff
@@ -809,12 +809,12 @@ Plural Format deals with this by breaking the problem into two parts:
 
```

Comment:
The link is not working : http://unicode.org/repos/cldr-tmp/trunk/diff/supplemental/language_plural_rules.html

The following is the error message:

Network Error (tcp_error)

> --

A communication error occurred: ""
The Web Server may be down, too busy, or experiencing other problems  preventing it from responding to requests. You may wish to try again at a  later time.
For assistance, contact your IT Helpdesk team at 1200 or itsupport.srib@samsung.com.Your request was categorized by Blue Coat Web Filter as 'Education;Technology/Internet'. If you wish to question or dispute this result, please click here.

Reply:
I checked the link sabeena21 reported, and it is also not working on my side.

To me, the message is like below.

---
The cldr-tmp repository is no longer available.

For access to CLDR sources and data, please see the [CLDR pages](http://cldr.unicode.org/).
---

Shouldn't we update the URL? For example to http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html?

Reply:
Changed link as proposed by mijn85cho